packages/content/src/notifications/items/PeerSharedAttributeDeletedByPeerNotificationItem.ts

```diff
@@ -0,0 +1,23 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { CoreId, ICoreId } from "@nmshd/transport";
+import { INotificationItem, NotificationItem, NotificationItemJSON } from "../NotificationItem";
+
+export interface PeerSharedAttributeDeletedByPeerNotificationItemJSON extends NotificationItemJSON {
+    "@type": "PeerSharedAttributeDeletedByPeerNotificationItem";
+    attributeId: string;
+}
+
+export interface IPeerSharedAttributeDeletedByPeerNotificationItem extends INotificationItem {
+    attributeId: ICoreId;
+}
+
+@type("PeerSharedAttributeDeletedByPeerNotificationItem")
+export class PeerSharedAttributeDeletedByPeerNotificationItem extends NotificationItem implements IPeerSharedAttributeDeletedByPeerNotificationItem {
+    @validate()
+    @serialize()
+    public attributeId: CoreId;
+
+    public static from(value: IPeerSharedAttributeDeletedByPeerNotificationItem): PeerSharedAttributeDeletedByPeerNotificationItem {
```

Comment:
`from(value: IPeerSharedAttributeDeletedByPeerNotificationItem | Omit<PeerSharedAttributeDeletedByPeerNotificationItemJSON, "@type">)`

Seems like we missed that for some of the notification items.